Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -3489,6 +3577,7 @@
 		08FB7794FE84155DC02AAC07 /* TestWebKitAPI */ = {
 			isa = PBXGroup;
 			children = (
+				3A5DDADC28D156AF004DA950 /* WebGPU.xcodeproj */,
```

Comment:
Assuming you're able to remove the explicit target dependency on WGTL, please also delete the project reference Xcode added. That will remove quite a bit of noise from the pbxproj diff :)